lightning/src/ln/channel.rs

```diff
@@ -3339,6 +3339,11 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 							channel_id: self.channel_id(),
 							data: "funding tx had wrong script/value".to_owned()
 						});
+					} else if txo_idx > 0xff_ff {
```

Comment:
Should these checks be tested? or a TODO, or issue to test them?

Reply:
Oh, oops, duh, funding_txo.index is a u16, so this line is unreachable.